### ISSUE TITLE
Requesting review of edits to the wiki Home page ("Getting started")

### ISSUE THREAD

**Issue by parke** [psy0rz/zfs_autobackup#119]
I copied the wiki Home page ("Getting started") into the wiki on my fork of zfs_autobackup.

I then made numerous edits to the Home page in my fork.

Do you have any comments or suggestions for improvements to my edits?

Would you like me to paste my changes into the Home page on your wiki?

I wanted to get your approval before making all these changes to your wiki.

You may view my edits at the following link:

https://github.com/parke/zfs_autobackup/wiki/Home/_compare/4864cec3b0e5fa567049ca1ed0bf136735184600...e3fcd0914160061567a66a38de3957201cc4571c

Thank you.

**Comment by psy0rz** [comment]
looks very good. and machine is indeed a more suitable term.

are wikis also forkable and pullable?

**Comment by parke** [comment]
> are wikis also forkable and pullable?

On the wiki page, there is a right hand column option:  
"Clone this wiki locally"  
With a link:  
https://github.com/psy0rz/zfs_autobackup.wiki.git

However, I don't know of anyway to clone or pull wiki content.

Regarding "machine".  I decided to standardize on a single term.  I believe you used "machine" first, but "server" elsewhere.  And I can imagine using zfs_autobackup on machines than are not really "servers" in any meaningful sense.  So I decided to use "machine".

I will cut and paste my edited page into your wiki.  Thanks!
